daffodil-core/src/main/scala/org/apache/daffodil/dsom/DFDLSchemaFile.scala

```diff
@@ -107,20 +107,15 @@ final class DFDLSchemaFile(val sset: SchemaSet,
     // parser throws out of fatalErrors.
   }
 
-  private lazy val loader = {
-    val ldr = new DaffodilXMLLoader(this)
-    // val shouldValidate = sset.validateDFDLSchemas
-    // ldr.setValidation(shouldValidate) // TODO: Validation not occurring JIRA DFDL-1473. Fix later.
-    ldr
-  }
-
   private def loadedNode = LV('loadedNode) {
     def die(e: Throwable) = {
       SDE("Error loading schema due to %s.", Misc.getSomeMessage(e).getOrElse("an unknown error."))
     }
     val node = try {
       log(LogLevel.Resolver, "Loading %s.", diagnosticDebugName)
-      val node = loader.load(schemaSource)
+      val ldr = new DaffodilXMLLoader(this)
+      ldr.setValidation(false)
```

Comment:
Does this also resolve DFDL-1473 mentioned above?

Reply:
Potentially, but this change is probably the cause of the regressions I didn't catch in sbt test. I'll see if this one can also be addressed.

Reply:
Yes. This resolves DAFFODIL-1473 also.